scripts/build-worker.sh

```diff
@@ -32,13 +32,17 @@ pushd "$(dirname "$0")/../"
     if [[ -n ${BUILD_TARGET_FILTER-} ]]; then
         BLOCK_MESSAGE="Building ${WORKER_TYPE} for ${BUILD_ENVIRONMENT} on ${BUILD_TARGET_FILTER} using ${SCRIPTING_BACKEND}"
         LOG_FILE="$(pwd)/../logs/${WORKER_TYPE}-${BUILD_ENVIRONMENT}-${BUILD_TARGET_FILTER}-${SCRIPTING_BACKEND}.log"
-        BUILD_TARGET_FILTER_ARG="+buildTargetFilter ${BUILD_TARGET_FILTER}"
+        BUILD_TARGET_FILTER_ARG="+buildTargetFilter ${BUILD_TARGET_FILTER} "
     else
         BLOCK_MESSAGE="Building ${WORKER_TYPE} for ${BUILD_ENVIRONMENT} using ${SCRIPTING_BACKEND}"
         LOG_FILE="$(pwd)/../logs/${WORKER_TYPE}-${BUILD_ENVIRONMENT}-${SCRIPTING_BACKEND}.log"
         BUILD_TARGET_FILTER_ARG=""
     fi
 
+    if [[ -n ${TARGET_IOS_SDK-} ]]; then
+        BUILD_TARGET_FILTER_ARG+="+targetiOSSdk ${TARGET_IOS_SDK}"
```

Comment:
Shouldn't this be its own argument? Rather than tagging onto `BUILD_TARGET_FILTER_ARG` ?